test/rating.js

```diff
@@ -16,40 +17,13 @@ describe('A track in Google Music', function () {
   });
   browserUtils.execute(function playViaApi () {
     window.googleMusic.playback.playPause();
+    window.googleMusic.playback.toggleRepeat(window.GoogleMusic.Playback.SINGLE_REPEAT);
```

Comment:
Can we add a comment before this line to indicate why we are doing the `repeat` setting?

```js
// DEV: Set to single repeat to prevent blocking other tests due to music stopping on no repeat
```